racket/collects/racket/private/hash.rkt

```diff
@@ -93,6 +93,22 @@
        [(hash-eqv? table) (make-hasheqv)]
        [(hash-eq? table) (make-hasheq)])]))
 
+  (define (hash-map/copy table f)
+    (unless (hash? table)
+      (raise-argument-error 'hash-map/copy "hash?" table))
+    (cond
```

Comment:
Should there be a  check here that `f` is a procedure that accepts 2 arguments?

Reply:
The contract in `racket/collects/racket/dict.rkt` has
```racket
 [dict-map/copy
  (->i ([d dict?]
        [proc (d) (-> (dict-key-contract d) (dict-value-contract d)
                      (values (dict-key-contract d) (dict-value-contract d)))])
       [_r dict?])]
```
But code that uses `racket/collects/racket/private/hash.rkt` directly will bypass that contract.

Reply:
Oh, sorry I got halfway through fixing it and my mind switched to the dict version. Yes, the hash version should have it, but it should be unnecessary for the dict version.

Reply:
I've updated `hash-map/copy`.